en/simulation/simulation-in-hardware.md

```diff
@@ -0,0 +1,37 @@
+# Simulation-In-Hardware (SIH)
+
+The Simulation-In-Hardware (SIH) is an alternative to the [Hardware In The Loop (HIL) simulation](../simulation/hitl.md) for a quadrotor. In this setup, the controller, the state estimator, and the simulator are running on the embedded hardware. Basically, everything is running on embedded hardware. The Desktop computer is used only to display the virtual quadrotor.
+
+![Simulator MAVLink API](../../assets/diagrams/SIH_diagram.png)
+
+The SIH provides two benefits over the HITL:
+- It ensures synchronous timing by avoiding the bidirectional connection to the computer. It means the user does not have to worry about having a powerful or real-time desktop computer.
+
+- The whole simulation remains inside the PX4 environment. It means developers familiar with PX4 willing to incorporate their own mathematical model into the simulator can do it easily. They can, for instance, modify the aerodynamic model, or noise level of the sensors, or even add a sensor to be simulated.
+
+The SIH can be used by new PX4 users to get familiar with PX4 and the different modes and features, and of course to learn to fly a quadrotor with the real Radio.
+
+The dynamic model is described in this [pdf report](../../assets/simulation/SIH_dynamic_model.pdf).
```

Comment:
Can you add the sources of that pdf as well?

Reply:
@bkueng 
This is currently a [google doc](https://docs.google.com/document/d/1ZAAkMnAsJGeEZe7MZiKhb7Pe8PKumlSzFmgxLU4qHEE/edit?usp=sharing)

Shall I add it as a .doc in /assets/simulation?

Reply:
A link to the doc is fine then. It's mostly in case someone finds mistakes/typos or changes some of the implementation, so that anyone can quickly update it.

Reply:
@romain-chiap The other alternative is to pull the doc into the guide itself. This is probably the best approach, but how about I do that as a post process (?) - equations and the wiki can be painful.

Reply:
Actually, decided it is OK as is.